inst/generate_examples.R

```diff
@@ -1,12 +1,5 @@
 source("tests/testthat/setup.R")
 set.seed(1)
-prep_obs <- as_latent_individual(sim_obs)
-fit <- epidist(prep_obs, seed = 1)
+
 saveRDS(fit, "inst/extdata/fit.rds")
-prep_obs_gamma <- as_latent_individual(sim_obs_gamma)
-fit_gamma <- epidist(
-  prep_obs_gamma,
-  family = stats::Gamma(link = "log"),
-  seed = 1
-)
 saveRDS(fit_gamma, "inst/extdata/fit_gamma.rds")
```

Comment:
Is there a point in having these saved if they are being generated in the test set-up file now?

Reply:
I don't know where else you are using them (i.e do you want to have them for examples etc so left for now)